### ISSUE TITLE
Quickcast settings not persistent between sessions

### ISSUE THREAD

**Issue by Izaaaaaa** [jeffhill/Dota2#1633]
### Description

Some heroes have non-persistent quickcast settings.

Setting shadow demon's poison (E) to quickcast, and the rest as normal (targetted) cast works fine. When quitting and restarting dota, this configuration is lost and reset to the default of targetted cast for all.

**Comment by Geradot** [comment]
Yes, the problem is still relevant. That is, if on some hero you mix up ability keys in the "CAST" and "QUICKCAST" tabs, then the "QUICKCAST" keys will work only until the end of the current match. For the next match, these keys will no longer be assigned to either "CAST" or "QUICKCAST", so you have to assign keys to "QUICKCAST" again.

Moreover, if on some hero all abilities are assigned to "QUICKCAST", then they are not reset after the match.
